google/datalab/contrib/mltoolbox/commands/_ml.py

```diff
@@ -291,6 +291,26 @@ def ml(line, cell=None):
   return google.datalab.utils.commands.handle_magic_line(line, cell, parser)
 
 
+def _abs_path(path):
+  """Convert a non-GCS path to its absolute path.
+
+  path can contain special filepath characters like '..', '*' and '.'.
+
+  Example: If the current folder is /content/datalab/folder1 and path is
+  '../folder2/files*', then this function returns the string
+  '/content/datalab/folder2/files*'.
+
+  This function is needed if using _shell_process.run_and_monitor() as that
+  function runs a command in a different folder.
+
+  Args:
+    path: string.
+  """
+  if not path.startswith('gs://'):
```

Comment:
nit (save a not, and generally avoid not in if condition):

if path.startswith('gs://'):
  return path

return os.path.abspath(path)